docs/design-language.md

```diff
@@ -7,6 +7,38 @@ import {ListControls} from "../src/components/menus";
 import {IconTabs, IndicatorColorTabs, NormalTabs, ScrollableTabs, VerticalTabs} from "../src/components/tabs";
 import {ButtonEmphasis, ButtonPairs, ButtonLabelsDo, ButtonLabelsDont, ButtonOptions, ButtonStyles, DisabledStates, ButtonColors, ButtonSizes, GroupedButtons, IconButtons, OutlinedButtons, ToggleButtons} from "../src/components/buttons";
 
+## Typograhphy
```

Comment:
Where in the order should we be placing the Typography section? Looks like we are generally using alphabetical order (the last three sections are out of order).

Reply:
I've re-ordered things, all should be in alphabetical order now.